unstructured/partition/pdf_image/pdfminer_processing.py

```diff
@@ -134,3 +137,55 @@ def merge_inferred_with_extracted_layout(
         inferred_page.elements[:] = elements
 
     return inferred_document_layout
+
+
+@requires_dependencies("unstructured_inference")
+def clean_pdfminer_inner_elements(document: "DocumentLayout") -> "DocumentLayout":
+    """Clean pdfminer elements from inside tables and stores them in extra_info dictionary
+    with the table id as key"""
+
+    from unstructured_inference.config import inference_config
+
+    defaultdict(list)
```

Comment:
Is there meant to be an assignment here? Just by itself this `defaultdict` will be garbage-collected, won't it?